src/browser-capabilities.ts

```diff
@@ -36,26 +38,31 @@ const browserPredicates: {
     es2015: since(49),
     push: since(41),
     serviceworker: since(45),
+    modules: since(61),
```

Comment:
Acknowledged. We can either add another capability for one or both of these, or just eventually say that "modules" means all of these things are supported.